android/app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
No, it didn't solve the simulator problem. Yes, it is necessary for using the gps hardware on the device.